runtime/parser2/parser.go

```diff
@@ -26,23 +26,31 @@ import (
 	"github.com/onflow/cadence/runtime/parser2/lexer"
 )
 
+// lowestBindingPower is the lowest binding power.
+// binding power decides the order of which expression to be parsed first.
+// the lower the binding power is, the latter the expression will be parsed.
```

Comment:
I'm not quite following this description. 

```suggestion
// The binding power controls operator precedence: 
// the higher the value, the tighter a token binds to the tokens that follow.
```